contrib/imgui/imgui_draw.cpp

```diff
@@ -119,40 +119,40 @@ static const ImVec4 GNullClipRect(-8192.0f, -8192.0f, +8192.0f, +8192.0f); // La
 
 void ImDrawList::Clear()
 {
-    CmdBuffer.resize(0);
```

Comment:
Did you just re-indent the imgui upstream file? :-)

That kinda makes the diff hard to read...